Demo/social/social_utils.py

```diff
@@ -0,0 +1,66 @@
+import csv
```

Comment:
What was the purpose of splitting social_demo.py to social_demo.py and social_utils.py?

Reply:
The reason I splited those 2 files is because I am using the populate function also in the tests.

Reply:
Sorry for the mess but I think it would be better if we join these two files (social_utils.py and social_demo.py) and simply have a "verify" flag which if on, will verify the query results and run time,
What do you think,
python social_demo.py -test ?

Reply:
I do agree that we shouldn't have files imdb_test and imdb_demo that do almost the same stuff, but still they are different. 
imdb_demo has the debug flag and it also prints all the responses from the queries, where the test only should check that the data and statistics are fine. Further more, the imdb_test should be in there the tests folder, because there sits all the tests and I don't think that the demo should be there because it doesn't look good. Meaning, we should have 2 different files that should sit in different folders and thats why we should have the utils file that will contain all the common functionality between imdb_demo and imdb_test.

Reply:
How would the test file get a hold of the utility file? (they are at a completely different folders)

Reply:
I use the sys.path.append command and tell it to find the file at "../../Demo/social"